### ISSUE TITLE
[BUG] <title>Can't create MAUI project on VSCode 1.84.1 on Mac 12.7

### ISSUE THREAD

**Issue by smartware163** [microsoft/vscode-dotnettools#699]
### Describe the Issue

VS Code Version: 1.84.1
OS Version: Mac 12.7

### Steps To Reproduce

Install VSCode and the MAUI extension
In the solution explorer click create .Net project, we just can't find the MAUI project template in the project template list.

**Comment by regetskcob** [comment]
For me on macOS 14.2 and VSCode 1.84.1 it's working, but i had installed Visual Studio for Mac as well, maybe this made the difference and installed something, which is missing on your side.

![Screenshot 2023-11-09 at 12 21 02](https://github.com/microsoft/vscode-dotnettools/assets/52231719/807a2473-01a8-4b8c-910a-bf1e2145c83e)

**Comment by smartware163** [comment]
I got the same error after upgrading Mac to 12.7.1 with only dotnet runtime 7.0.13
![Uploading 截屏2023-11-07 21.44.25.png…]()

**Comment by smitpatel** [comment]
@smartware163 - Can you run `dotnet new list maui` in your terminal in vscode and paste output here? It looks like either MAUI templates are not available on machine or C# dev kit is not able to find it based on dotnet being used.

**Comment by FullofQuarks** [comment]
I'm also running into this issue. Fresh MacOS 14 install. No Visual Studio for Mac installed, and I never installed anything dotnet before the VSCode C# dev kit. The plugin installed dotnet sdk 7.0.43. I then installed the maui vscode plugin. 

Output for `dotnet new list maui` returns

```bash
No templates found matching: 'maui'.

To search for the templates on NuGet.org, run:
   dotnet new search maui


For details on the exit code, refer to https://aka.ms/templating-exit-codes#103
```

**Comment by smitpatel** [comment]
@kvenkatrajan - Who should we ping from MAUI extension? I am not entirely sure who is supposed to install maui workload and relevant templates in this case.

**Comment by smartware163** [comment]
> I'm also running into this issue. Fresh MacOS 14 install. No Visual Studio for Mac installed, and I never installed anything dotnet before the VSCode C# dev kit. The plugin installed dotnet sdk 7.0.43. I then installed the maui vscode plugin.
> 
> Output for `dotnet new list maui` returns
> 
> ```shell
> No templates found matching: 'maui'.
> 
> To search for the templates on NuGet.org, run:
>    dotnet new search maui
> 
> 
> For details on the exit code, refer to https://aka.ms/templating-exit-codes#103
> ```

I have the same output

**Comment by lorenzosjb** [comment]
Found out by accident reading https://devblogs.microsoft.com/visualstudio/announcing-the-dotnet-maui-extension-for-visual-studio-code/, that using the command `dotnet install workload maui` installed the templates

**Comment by smitpatel** [comment]
I am removing area-templates label for now. The scenario works if maui workload is installed. The error here is that maui workload is not present even if MAUI extension installed.

**Comment by FullofQuarks** [comment]
Yes, the workload is missing, this is why the templates cannot be found. However, a smooth developer experience is always nice. The expected experience would be similar to that of the c# dev kit. While the dev kit is installing, it installs everything that it needs to function, such as dotnet. 

The experience could be similar in that the workload is installed on install of the MAUI extension

**Comment by smartware163** [comment]
After executing dotnet workload install maui in the terminal, the MAUI templates appear and we can create related projects.

**Comment by jknaudt21** [comment]
Although workload management and project creation will remain tied to C# Dev Kit extension, we've added a warning for users trying to create a new project. The warning notes that maui workload is not available if they have the Maui extension installed.

**Comment by iq007** [comment]
> `dotnet install workload maui`

Should be 'dotnet workload install maui'. You need elevated privileges.

**Comment by iq007** [comment]
> Found out by accident reading https://devblogs.microsoft.com/visualstudio/announcing-the-dotnet-maui-extension-for-visual-studio-code/, that using the command `dotnet install workload maui` installed the templates.
> 
> More at https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code#install-visual-studio-code-and-the-net-maui-extension

Should be 'dotnet workload install maui'. Run with elevated privileges.

**Comment by Prog-NET-mobile** [comment]
In Visual Studio Code open terminal (View/Terminal) and run commands:
`$ sudo dotnet workload install maui
$ dotnet new install Microsoft.Maui.Templates::6.0.312`
Restart VS Code, now will be possible to create MAUI project.
